modules/cnf-configuring-node-groups-for-the-numaresourcesoperator.adoc

```diff
@@ -17,7 +17,7 @@ The configuration options are the following:
 +
 [NOTE]
 ====
-`podsFingerprinting` is enabled by default. `podsFingerprinting` is a requirement for the `cacheResyncPeriod` specification in the `NUMAResourcesScheduler` CR. The `cacheResyncPeriod` specification helps to report more exact resource availability by monitoring pending resources on nodes.
+The default value for `podsFingerprinting` is `EnabledExclusiveResources`. You need to set `podsFingerprinting` to `Enabled` for the `cacheResyncPeriod` specification in the `NUMAResourcesScheduler` CR. The `cacheResyncPeriod` specification helps to report more exact resource availability by monitoring pending resources on nodes. test
```

Comment:
I'm not completely sure whether EnabledExclusiveResources would also be enough to enable setting the `cacheResyncPeriod` specification in the `NUMAResourcesScheduler` CR or if it is only via Enabled. @ffromani, may you share your input here?

Reply:
`podsFingerprinting` accepts three values: `Disabled`, `Enabled`, `EnabledExclusiveResources`. For the scheduler to work optimally, either `Enabled` or `EnabledExclusiveResources` must be enabled (the latter is the default). At the same time, for the scheduler to work optimally, `cacheResyncPeriod` must be set to any value > 0. So we need **both** `podsFingerPrinting` and `cacheResyncPeriod` to work in concert, and the two values can be set inconsistently (we can't prevent bad configurations yet).

The `podsFingerprinting` value is meant for special tunings though. Users are not expected to change it except for exceptional circumstances